### PR TITLE
Fix CSS issues

### PR DESCRIPTION
**Pull request by FinalAngel**
- correct header positioning/alignment
- move header when toolbar is expanding (properly)
- fix font on accordion
- separate cms addons to own file
- adapt settings for remaining bootstrap components
- update boilerplate files

### CODE REVIEW

File: templates/includes/menu/langnav.html [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
 
 {% if languages|length > 1 %}
     <li class="child lang dropdown">
-        <a href="{% page_language_url lang.code %}" class="dropdown-toggle" data-toggle="dropdown"><span>{{ lang.code|language_name_local|title }} <span class="caret"></span></span></a>
+        <a href="{% page_language_url lang.code %}" class="dropdown-toggle" data-toggle="dropdown"><span>{{ lang.name }} <span class="caret"></span></span></a>
         <ul class="dropdown-menu" role="menu">
             {% for language in languages %}
                 {% if not language.0 == current_language %}
-                    <li class="child lang"><a href="{% page_language_url language.0 %}"><span>{{ language.0|language_name_local|title }}</span></a></li>
+                    <li class="child lang"><a href="{% page_language_url language.0 %}"><span>{{ language.1 }}</span></a></li>
```
Comment (vxsx) [inline templates/includes/menu/langnav.html]: ?

File: templates/includes/menu/langnav.html [hunk 2]
```diff
@@ -1,14 +1,13 @@
 {% load i18n menu_tags %}
 {% get_language_info for LANGUAGE_CODE as lang %}
-{# DOCS: https://docs.djangoproject.com/en/1.5/topics/i18n/translation/#other-tags #}
 
 {% if languages|length > 1 %}
     <li class="child lang dropdown">
-        <a href="{% page_language_url lang.code %}" class="dropdown-toggle" data-toggle="dropdown"><span>{{ lang.code|language_name_local|title }} <span class="caret"></span></span></a>
+        <a href="{% page_language_url lang.code %}" class="dropdown-toggle" data-toggle="dropdown"><span>{{ lang.name }} <span class="caret"></span></span></a>
```
Comment (vxsx) [inline templates/includes/menu/langnav.html]: why change this?

Comment (FinalAngel) [inline templates/includes/menu/langnav.html]: overwrite from the boilerplate

**Comment by vxsx** [comment]
LGTM

**Pull request merged**